packages/sdk-codegen/src/python.gen.ts

```diff
@@ -536,6 +536,7 @@ ${this.hooks.join('\n')}
       number: { default: this.nullStr, name: 'float' },
       password: { default: this.nullStr, name: 'str', asVal: asString },
       string: { default: this.nullStr, name: 'str', asVal: asString },
+      object: { default: this.nullStr, name: 'str', asVal: asString },
```

Comment:
@drstrangelooker what necessitated this change?

Reply:
There is one place where an object is part of the query string, the `filters` parameter of `"/models/{model_name}/views/{view_name}/fields/{field_name}/suggestions"`. It results in a reference to `mdls.object` which doesn't exist.